interface/main/tabs/main.php

```diff
@@ -222,13 +221,39 @@ function jsFetchGlobals(scope) {
         })
     })
 }
-
 jsFetchGlobals('top').then(globalJson => {
     jsGlobals = globalJson;
 }).catch(error => {
     console.log(error.message);
 });
 
+// set up global translations for js
+function setupI18n(lang_id) {
+    top.restoreSession();
+    return new Promise((resolve, reject) => {
+        fetch(<?php echo js_escape($GLOBALS['webroot'])?> + "/library/ajax/i18n_generator.php?lang_id=" + encodeURIComponent(lang_id), {
+            credentials: 'same-origin',
+            method: 'GET'
+        })
+        .then(response => response.json())
+        .then(data => resolve(data))
+        .catch(error => reject(error))
+    })
+}
+setupI18n(<?php echo js_escape($_SESSION['language_choice']); ?>).then(translationsJson => {
+    i18next.init({
+        lng: 'selected',
+        debug: false,
+        resources: {
+            selected: {
+                translation: translationsJson
+            }
+        }
+    });
+}).catch(error => {
+    console.log(error.message);
+});
```

Comment:
@sjpadgett , Hopefully I did this right :) (it does work on testing)

Reply:
There was nothing wrong with the way you originally did this but now we can grab the translated object from anywhere if somebody decides they need it.
Once this is in I decided to go ahead and remove the jsFetchGlobals() and rework where it is used in dialog.js to new js translations.

Reply:
Do note this PR is leveraging jsGlobals for one of it's translation choices. (line 334 in interface/main/tabs/js/tabs_view_model.js )